packages/mui-material/src/StepLabel/StepLabel.js

```diff
@@ -112,6 +111,9 @@ const StepLabelLabelContainer = styled('span', {
 })(({ theme }) => ({
   width: '100%',
   color: (theme.vars || theme).palette.text.secondary,
+  [`&.${stepLabelClasses.alternativeLabel}`]: {
```

Comment:
What do you think about adding font family here?
```suggestion
  fontFamily: theme.typography.fontFamily,
  [`&.${stepLabelClasses.alternativeLabel}`]: {
```

Reply:
I saw that the `Optional` text has a default font family.

Reply:
Yes, we could.

Reply:
It's a breaking change. Argos detects changes: https://app.argos-ci.com/mui/material-ui/builds/5315. I'm reverting it.